honeybee_radiance/exception.py

```diff
@@ -0,0 +1,61 @@
+"""Honeybee radiance exceptions."""
```

Comment:
Very cool thing you did here.  We might consider something similar for all of the exceptions that can get raised from the checking methods in honeybee-core.

Reply:
That can be. Custom exceptions are definitely helpful but I think we should just add them if the normal exceptions are not serving our purpose. Otherwise we can end up with one more thing to maintain and for no good reason.